.circleci/config.yml

```diff
@@ -3,23 +3,42 @@ version: 2.1
 workflows:
   test:
     jobs:
-      - build-run-linux
+    - build-run-linux:
+        context: hello-world-demos
+
+  test-daily:
+    triggers:
+    - schedule:
+        cron: "0 6 * * *"
+        filters:
+          branches:
+            only: master
+    jobs:
+    - build-run-linux:
+        context: hello-world-demos
 
 # This CI build ensures that the demo both compiles and works correctly. For the runtime test,
-# we use an SDK key and flag key that are passed in via the CircleCI project configuration;
-# the flag is configured to return a true value.
+# we use an SDK key and flag key that are passed in via the CircleCI context; the flag is
+# configured to return a true value.
 
 jobs:
   build-run-linux:
     docker:
-      - image: circleci/golang:1.14
+      - image: cimg/go:1.14
```

Comment:
Unrelated - this is just the newer Docker tag for the standard CircleCI Go image.